### ISSUE TITLE
Kirby 3 - Make sure cache works

### ISSUE THREAD

**Issue by jenstornell** [jenstornell/kirby-html-minifier#18]
Use component instead

**Comment by bavarianbytes** [comment]
Does the plugin support Kirby 3?

**Comment by jenstornell** [comment]
> Does the plugin support Kirby 3?

No, sorry. It should probably be quite easy to convert it to Kirby 3 for someone who knows how it's done, but I don't have the time, sorry.
